NEWS.md

```diff
@@ -13,6 +13,13 @@
 * `oauth_flow_refresh()` now only warns if the `refresh_token` changes, making
   it a little easier to use in manual workflows (#186).
 
+* `oauth_flow_auth_code()` now attempts to detect when a temporary webserver can
+  be run on `localhost`. If not, it allows users to enter the authorisation code
```

Comment:
I think it would be good to be explicit about Collab/Workbench/Cloud since otherwise this sounds like it detects if you cant start a web server.